shared/chat/conversation/list-area/normal/index.desktop.tsx

```diff
@@ -591,115 +588,114 @@ class OrdinalWaypoint extends React.Component<OrdinalWaypointProps, OrdinalWaypo
   // we defer settings things invisible for a little bit, which seems enough to fix it
   private handlePositionChange = p => {
     // lets ignore when this happens, this seems like a large source of jiggliness
-    if (this.state.isVisible && !p.event) {
+    if (this.isVisible && !p.event) {
       return
     }
     const {currentPosition} = p
     if (currentPosition) {
       const isVisible = currentPosition === 'inside'
```

Comment:
it might be nice to change this variable name to something besides `isVisible` so there's not any confusion with `this.isVisible`

Reply:
good idea, renamed